sources/@roots/bud-typescript/eslint-config/index.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
```

Comment:
eslint-config for ts: `@roots/bud-typescript/eslint-config`.

This will be the first thing that breaks if a user tries typescript.